doc/source/io.rst

```diff
@@ -136,7 +136,11 @@ usecols : array-like or callable, default ``None``
   that correspond to column names provided either by the user in `names` or
   inferred from the document header row(s). For example, a valid array-like
   `usecols` parameter would be ``[0, 1, 2]`` or ``['foo', 'bar', 'baz']``. 
-  Element order is ignored, so ``usecols=[0, 1]`` is the same as ``[1, 0]``.
+  Element order is ignored, so ``usecols=[0, 1]`` is the same as ``[1, 0]``. Element
+  order is ignored, so usecols=[1,0] is the same as [0,1]. To instantiate a
+  DataFrame with element order preserved use ``pd.read_csv(usecols=[0, 1])[[0, 1]]``
```

Comment:
it should be `“pd.read_csv(usecols=[0, 1])[[1, 0]]”` ？

Reply:
Yep.

Reply:
Thanks for catching that.   Fixed in commit 02715087b61b73c6142abae75b4272018eb07402